### DISCUSSION TITLE
Support for JWKS

### DISCUSSION THREAD

**Discussion by andriisly** [nanomq/nanomq#1913] [Q&A]
Hello NanoMQ team,

We are using EMQX in the Cloud as our MQTT broker and it supports [JWKS](https://docs.emqx.com/en/emqx/latest/access-control/authn/jwt.html#authentication-principle) to verify tokens by periodically fetching the latest keys from the JWKS endpoint. We are considering using the NanoMQ broker on an On-Premise solution on a Windows 2016 server, but I could not find any information on whether NanoMQ supports JWKS.

Could you please confirm whether NanoMQ supports JWKS and if not, if there are any plans to implement this feature in future releases? Maybe you know of other ways to use JWKS with NanoMQ?

**Comment by JaylinYu** [comment]
For now, HTTP server of nanomq only supports JWT nor JWKS.
